src/app/about/page.tsx

```diff
@@ -0,0 +1,142 @@
+import Head from "next/head";
+import Image from "next/image";
+import { headers } from "next/headers";
+import Init_Team from "./Init_Team";
+import Chapter_card from "./Chapter_card";
+import { Chapter } from "../api/chapters/route";
+import { Member } from "../api/eboard/route";
+
+export default async function About() {
+  const origin = headers().get("x-origin");
+  const chaptersRequest = await fetch(`${origin}/api/chapters`);
+  const chapters: Chapter[] = await chaptersRequest.json();
+  const eboardMembersRequest = await fetch(`${origin}/api/eboard`);
+  const team_members: Member[] = await eboardMembersRequest.json();
+  console.log(team_members);
+  return (
+    <>
+      <section className="mx-auto max-w-screen-lg ">
+        <Head>
+          <title>About Init</title>
+          <meta name="description" content="About us for Init" />
+        </Head>
+        <main className="flex flex-col bg-primary"></main>
+        <h1 className="mb-10 text-center text-4xl text-white">💻 About Us</h1>
+        <br />
+        <span className=" flex flex-wrap  ">
+          <div className="mx-auto mb-10 h-auto w-full text-sm  ">
+            <div className="mb-5">
+              <iframe
+                className=""
+                src="https://www.youtube.com/embed/q44VgsbcBkw"
+                title="INIT Launch Announcement"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowFullScreen={true}
+                height={500}
+                style={{ width: "100%" }}
+              ></iframe>
+            </div>
+          </div>
+        </span>
+        <br />
+
+        <h1 className="mb-10 text-center text-4xl text-white">❤️Impact</h1>
+        <br />
+        <div className="mx-auto mb-10 flex h-auto w-auto flex-wrap items-start justify-center ">
+          <Image
+            src="/Empowering_tech_talent.jpg"
```

Comment:
Not the correct path to the image, correct path is: /images/eboard/Empowering_tech_talent.jpg